OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/12.2.0.1/MultiRACInstall.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
This script needs both a copyright and a 1-line UPL license header.

Reply:
Removed the file as at this moment it is not required.